scripts/download.sh

```diff
@@ -65,6 +69,23 @@ if [ ! -f /var/spool/ctlstore/ldb.db ]; then
   fi
   SHA_END=$(date +%s)
   echo "Local checksum calculation took $(($SHA_END - $SHA_START)) seconds"
+}
+
+if [ ! -f /var/spool/ctlstore/ldb.db ]; then
+  mkdir -p /var/spool/ctlstore
+  cd /var/spool/ctlstore
+
+  download_snapshot
+  check_sha
+
+  i=2
+  while [ "$i" -le $NUM_LDB ]; do
+    if [ ! -f ldb-$i.db ]; then
+      echo "creating copy ldb-$i.db"
+      cp snapshot.db ldb-$i.db
```

Comment:
Looking at [Disk Utilization per Host](https://segment.datadoghq.com/dashboard/xun-45u-377/integrations-consumer?fullscreen_end_ts=1700533071575&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1700529471575&fullscreen_widget=6868805985850044&refresh_mode=sliding&tile_focus=6868805985850044&tpl_var_app%5B0%5D=integrations-consumer&tpl_var_env=production&from_ts=1700528745273&to_ts=1700532345273&live=true) some of the int-consumer nodes are already having limited headroom left, with current size of the ldb, this change will incur extra 16G space usage if my understanding of the file system is correct. 
<img width="1868" alt="Screenshot 2023-11-20 at 9 07 39 PM" src="https://github.com/segmentio/ctlstore/assets/6007569/9caa2da8-a25b-4ed8-a189-cbc75229ca25">

Reply:
it's likely we'll have to carefully roll this out. The disk is so close to full because of the large WAL files. If we have 2 ldbs and a snapshot, we'll have a moment during the init with 24 GB of files. But under normal circumstances, this won't be a concern. Looking at the replay consumers gives you a better idea of how much headroom the nodes have under normal ldb performance

Reply:
Ah, with a corrected understanding of the WAL file, I don't think the chart fairly reflected the disk usage of int-consumer nodes as it doesn't factor in the new introduction of the daily restarting cron job. I will rotate those nodes that have high disk usage next Wed after HAP ends, after which disk space would no longer likely to be a concern anymore.